doc-content/jbpm-docs/src/main/asciidoc/BPMN2/bpmn-subprocesses-ref.adoc

```diff
@@ -0,0 +1,74 @@
+* *Is Async*: Select if the task should be invoked asynchronously and cannot be executed instantly.
+* *Multiple Instance*: Select to execute the subprocess elements a specified number of times. If selected, the following options are available:
+** *MI Execution mode*: Indicates if the multiple instances execute in parallel or sequentially. If set to *Sequential*, new instances are not created until the previous instance completes.
+** *MI Collection input*: Select a variable that represents a collection of elements for which new instances are created. The subprocess is instantiated as many times as the size of the collection.
+** *MI Data Input*: Specifies the name of the variable containing the selected element in the collection. The variable is used to access elements in the collection.
+** *MI Collection output*: Optional variable that represents the collection of elements that will gather the output of the multi-instance node.
+** *MI Data Output*: Specifies the name of the variable that is added to the output collection that you selected in the *MI Collection output* property.
+** *MI Completion Condition (mvel)*: MVEL expression that is evaluated on each completed instance to check if the specified multiple instance node can complete. If it evaluates to `true`, all remaining instances are canceled.
+* *On Entry Action*: A Java or MVEL script that specifies an action at the start of the task.
+* *On Exit Action*: A Java or MVEL script that specifies an action at the end of the task.
+* *SLA Due Date*: The date that the service level agreement (SLA) expires.
+
+.Reusable subprocess properties
+image::BPMN2/subproc-props.png[A screenshot of Subprocess properties]
+
+[[_event_sub_process]]
+.Event subprocess
+An event subprocess becomes active when its start event is triggered. It can interrupt the parent process context or run in parallel with it.
+
+With no outgoing or incoming connections, only an event or a timer can trigger the subprocess. The subprocess is not part of the regular control flow.
+Although self-contained, it is executed in the context of the bounding process.
+
+Use an event subprocess within a process flow to handle events that happen outside of the main process flow.
+For example, while booking a flight, two events may occur:
+
+* Cancel booking (interrupting)
+* Check booking status (non-interrupting)
+
+You can model both of these events using the event subprocess.
+
+[[_multiple_instances]]
+.Multiple instance subprocess
+
+A multiple instances subprocess is instantiated multiple times when its execution is triggered. The instances are created sequentially. A new subprocess instance is created only after the previous instance has finished.
+
+A multiple instances subprocess has one incoming connection and one outgoing connection.
```

Comment:
@MarianMacik Is multi instance subprocess is a part of embedded subprocess? In Business Central, embedded subprocess and multi instance subprocess are two individual subprocesses. Please correct if I am wrong. 
![image](https://user-images.githubusercontent.com/59050394/101473509-ea4d0000-396f-11eb-82ac-45de8dbfb7ed.png)

Reply:
I read it once again and probably it can be left as it is now. So we have
* Reusable subprocess
where one configuration option is to run it as a multiple instance one.
* Multiple instances subprocess
which is always run like that and acts as an embedded subprocess in multiple instance mode.

Reply:
@MarianMacik got it, thanks